examples/testerina-mocking-objects/testerina_mocking_objects_test.out

```diff
@@ -5,9 +5,6 @@ ballerina test bbe_mocking
 Compiling source
         ballerinatest/bbe_mocking:0.1.0
 
-Creating balos
```

Comment:
@Ibaqu if you are considering this to a single file test execution, we might need to update the package details under "compiling source" section. Can you verify this?

Reply:
Verified with the latest pack. `Creating balos` has been removed.